tests/pfcwd/test_pfcwd_timer_accuracy.py

```diff
@@ -164,26 +165,27 @@ def run_test(self):
         with DisableLogrotateCronContext(self.dut):
             logger.info("Flush logs")
             self.dut.shell("logrotate -f /etc/logrotate.conf")
-        self.storm_handle.start_storm()
-        logger.info("Wait for queue to recover from PFC storm")
-        time.sleep(16)
-        if self.dut.topo_type == 't2' and self.storm_handle.peer_device.os == 'sonic':
-            storm_detect_ms = self.retrieve_timestamp("[d]etected PFC storm")
+        # Retry logic as workaround for issue #10848
+        num_tries = 3
+        for _ in range(num_tries):
+            storm_start_time_ms = time.time() * 1000
```

Comment:
storm_start_time_ms value would be reset in every loop, is it the expected behavior?

Reply:
Yes, intention of this is to record the start time for a storm that is actually detected.

Reply:
I remember every start_storm() would output the message "PFC_STORM_START".
If we have to retry, please consider the timestamp, which message's timestamp we need to catch, thanks